assets/setup-wizard/style.scss

```diff
@@ -21,6 +21,11 @@
 		.sensei-setup-wizard__learn-more {
 			white-space: nowrap;
 		}
+
+		.sensei-setup-wizard__woocommerce-observation {
+			display: block;
```

Comment:
Can get rid of this if it's a `<div>` instead of `<span>`. Also wrapping it with `<em>` should be a good fit here, semantics-wise.

Reply:
Yeah! It bothered me a little too, but it's using a inline element because the list is mounted with span. So otherwise we'd have block elements inner inline elements :(

But the `em` makes sense! Updated the `span` to `em` here: https://github.com/Automattic/sensei/pull/3297/commits/a6ae1e56fd0ef7b6e01cff09b341cf312b9b079a

Anyway, I kept the style to add the italic, because we don't have any style for em and it would come from the browser styles.